async_dynamic_log/src/log_dynamic.mli

```diff
@@ -0,0 +1,6 @@
+module Global_dynamic : sig
+  open! Async
```

Comment:
Same thing here. Let's have our `open` statements outside this signature, and at the top of the file. We should also always import `Core`.